test/graphql/queries/profile_query_test.rb

```diff
@@ -5,7 +5,7 @@
 class ProfileQueryTest < ActiveSupport::TestCase
   setup do
     users(:test).update account: accounts(:test)
-    profiles(:one).update account: accounts(:test), hosts: [hosts(:one)]
+    profiles(:one).update account: accounts(:test)
```

Comment:
Hosts are associated via policy. The previous case would associate hosts with a profile through a test result.